data/registry/collector-exporter-oracle-database.yml

```diff
@@ -0,0 +1,18 @@
+title: Oracle Database Metrics Exporter
+registryType: exporter
+language: go
```

Comment:
this would mean that it is an exporter that go developers can use, but if I understand it correctly this is a standalone tool, right?

Reply:
```
registryType: utilities
language: go
```

in that case. Also rename the file to: `tools-oracle-database-metrics-exporter`

Reply:
but it's not an OpenTelemetry collector, right? Only in that case the registryType can be "collector"